.github/workflows/release-backwards-compatiblity.yml

```diff
@@ -21,10 +21,22 @@ env:
   MESSAGING_USER_ARN: ${{secrets.MESSAGING_USER_ARN}}
 
 jobs:
+  check-backward-compatibility-run:
+    name: Check need for backward compatibility checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Version check
+        run: node .github/script/backward-compatibility-run-check.js
+
   release-integ-test:
     name: Run Backwards Compatible Integration Tests with Release Tarball
     runs-on: ubuntu-latest
-
+    needs: check-backward-compatibility-run
+    if: ${{ success() }}
```

Comment:
Sorry not sure we want this as it will show the workflow as fail right (as the previous job fail? )

Reply:
Discussed offline, but did this: https://github.com/aws/amazon-chime-sdk-js/pull/2077#discussion_r817968614